### DISCUSSION TITLE
Asking the main developer about the bug tracker (some questions)

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [black7375/Firefox-UI-Fix#679] [Q&A] [answered]
— Have you ever created a meta bug?
— Have you ever assigned someone different (I think someone who is not developer of Firefox UI Fix) into bugs?
— How are you setting the severity of bugs?
— Correct me if I'm wrong: You are setting priorities through tags.
— Have you ever deleted some issues (Spam, advertisement, abuse)?

I will thank for any response, and sorry if I'm annoying.

**Comment by black7375** [comment, marked as answer]
1. Have you ever created a meta bug?
  I don't know what the purpose of this question is, but there is.
  It's a typical example #175 
  Feedback threads can also be seen as one of the meta bug tracker #351 #438 #511
2. Have you ever assigned someone different (I think someone who is not developer of Firefox UI Fix) into bugs?
  Not yet.
  But @2641a40fd44383320adde4b027a1d0b03bd550 is handling the rounding issue of this project well. Thanks!! 
3. How are you setting the severity of bugs?
  Features used by many people, such as the default option, are priority.
4. Correct me if I'm wrong: You are setting priorities through tags.
  As a user, it is natural not to know all internal information or development details, so you don't have to worry about it if it's wrong.
5. Have you ever deleted some issues (Spam, advertisement, abuse)?
  I don't know the intention of the question, but..
  There was a spam several times, but I reported it to GitHub and processed. (There issues are deleted by GitHub)
  There was an example of creating a commit by stealing other people's emails. #60
  This was an opportunity to specify the [code of conduct](https://github.com/black7375/Firefox-UI-Fix/blob/master/CODE_OF_CONDUCT.md).

**Reply by Aleksandra0056** [reply]
You didn't understood the fourth question, I think marking priorities of bugs with tags. 

E.g. Fatal, Critical, Major, Medium, Minor, Low, Very Low.

**Reply by black7375** [reply]
To be exactly, I haven't got motivated to express importance on all issues with tags yet.

**Reply by Aleksandra0056** [reply]
So you think, that it would be useless to have tags like Priority::Major, or Priority::Fatal?

**Reply by Aleksandra0056** [reply]
Oh sorry, I didn't understand. Well, someone would say: “Hey! My issue is not important as others issues?!” 

Now I understood right, yeah?

**Reply by black7375** [reply]
Yes, that's right, it matches my thoughts.
